src/vs/base/parts/sandbox/electron-browser/preload.js

```diff
@@ -68,7 +68,12 @@
 		}
 
 		try {
-			if (validateIPC(windowConfigIpcChannel)) {
+			/**
+			 * Only want to call ipcRenderer.invoke when the IPC channel isn't for testing because it causes ADS scrollable
+			 * view tests to timeout and fail. This could be a result of the way the basic 'ISandboxConfiguration' is setup
```

Comment:
Do you know why calling invoke causes the tests to not run?

Reply:
No, I don't.

I was able to isolate the problem to this call after I copied over the version of preload.js before the VS Code merge that broke the scrollable view tests, and comparing the differences between those two versions of the same file and seeing what happened when I commented out the changes that came in after the merge.